<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/res/462574542_1124823785684518_3296515449835166663_n.png);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>871</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/res/462574542_1124823785684518_3296515449835166663_n.png);</string>
    </property>
    <property name="title">
     <string>PRODUITS</string>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>951</width>
       <height>501</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(1, 1, 1);</string>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::TabPosition::North</enum>
     </property>
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Ajout</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit_Id_F_P">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>120</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Name">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>170</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Prix">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>230</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonAjouter">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>410</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Categorie">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>290</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Quantite">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>340</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>300</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid rgb(255, 165, 0); /* Couleur de la bordure orange */
background-color: rgb(255, 255, 255);/* Couleur d'arrière-plan */
border-radius: 10px; /* Coins arrondis */
color: rgb(0, 0, 0); /* Couleur du texte en noir */
</string>
       </property>
       <property name="text">
        <string>Categorie:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lineEditProductId_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid rgb(255, 165, 0); /* Couleur de la bordure orange */
background-color: rgb(255, 255, 255);/* Couleur d'arrière-plan */
border-radius: 10px; /* Coins arrondis */
color: rgb(0, 0, 0); /* Couleur du texte en noir */
</string>
       </property>
       <property name="text">
        <string>Id fournisseur :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>350</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid rgb(255, 165, 0); /* Couleur de la bordure orange */
background-color: rgb(255, 255, 255);/* Couleur d'arrière-plan */
border-radius: 10px; /* Coins arrondis */
color: rgb(0, 0, 0); /* Couleur du texte en noir */
</string>
       </property>
       <property name="text">
        <string>Quantite du stock :</string>
       </property>
      </widget>
      <widget class="QLabel" name="lineEditPrice_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid rgb(255, 165, 0); /* Couleur de la bordure orange */
background-color: rgb(255, 255, 255);/* Couleur d'arrière-plan */
border-radius: 10px; /* Coins arrondis */
color: rgb(0, 0, 0); /* Couleur du texte en noir */</string>
       </property>
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
      <widget class="QLabel" name="lineEditProductName_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid rgb(255, 165, 0); /* Couleur de la bordure orange */
background-color: rgb(255, 255, 255);/* Couleur d'arrière-plan */
border-radius: 10px; /* Coins arrondis */
color: rgb(0, 0, 0); /* Couleur du texte en noir */
</string>
       </property>
       <property name="text">
        <string>Nom  de produit </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Id_P">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>70</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="lineEditProductId_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid rgb(255, 165, 0); /* Couleur de la bordure orange */
background-color: rgb(255, 255, 255);/* Couleur d'arrière-plan */
border-radius: 10px; /* Coins arrondis */
color: rgb(0, 0, 0); /* Couleur du texte en noir */
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string>Id produit :</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::MarkdownText</enum>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Affichage</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>30</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonAfficher">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>121</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Afficher Tout</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>90</y>
         <width>701</width>
         <height>281</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonRecherche">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>30</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Afficher Selon Categorie</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonPDF">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>400</y>
         <width>121</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Exporter PDF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonTrierType">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>400</y>
         <width>151</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Trier selon categorie</string>
       </property>
      </widget>
      <widget class="QPushButton" name="generatetypeButton">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>400</y>
         <width>221</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>statistique selon categorie</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonTrierPrix">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>400</y>
         <width>151</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Trier selon prix</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Modification</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>40</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonModifier">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>400</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QLabel" name="lineEditProductId_5">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid rgb(255, 165, 0); /* Couleur de la bordure orange */
background-color: rgb(255, 255, 255);/* Couleur d'arrière-plan */
border-radius: 10px; /* Coins arrondis */
color: rgb(0, 0, 0); /* Couleur du texte en noir */
</string>
       </property>
       <property name="text">
        <string>Id produit </string>
       </property>
      </widget>
      <widget class="QLabel" name="lineEditProductId_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid rgb(255, 165, 0); /* Couleur de la bordure orange */
background-color: rgb(255, 255, 255);/* Couleur d'arrière-plan */
border-radius: 10px; /* Coins arrondis */
color: rgb(0, 0, 0); /* Couleur du texte en noir */
</string>
       </property>
       <property name="text">
        <string>Id fournisseur </string>
       </property>
      </widget>
      <widget class="QLabel" name="lineEditProductName_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>160</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid rgb(255, 165, 0); /* Couleur de la bordure orange */
background-color: rgb(255, 255, 255);/* Couleur d'arrière-plan */
border-radius: 10px; /* Coins arrondis */
color: rgb(0, 0, 0); /* Couleur du texte en noir */
</string>
       </property>
       <property name="text">
        <string>Nom  de produit </string>
       </property>
      </widget>
      <widget class="QLabel" name="lineEditPrice_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>220</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid rgb(255, 165, 0); /* Couleur de la bordure orange */
background-color: rgb(255, 255, 255);/* Couleur d'arrière-plan */
border-radius: 10px; /* Coins arrondis */
color: rgb(0, 0, 0); /* Couleur du texte en noir */</string>
       </property>
       <property name="text">
        <string>Prix </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>280</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid rgb(255, 165, 0); /* Couleur de la bordure orange */
background-color: rgb(255, 255, 255);/* Couleur d'arrière-plan */
border-radius: 10px; /* Coins arrondis */
color: rgb(0, 0, 0); /* Couleur du texte en noir */
</string>
       </property>
       <property name="text">
        <string>Categorie </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>350</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid rgb(255, 165, 0); /* Couleur de la bordure orange */
background-color: rgb(255, 255, 255);/* Couleur d'arrière-plan */
border-radius: 10px; /* Coins arrondis */
color: rgb(0, 0, 0); /* Couleur du texte en noir */
</string>
       </property>
       <property name="text">
        <string>Quantite du stock </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>100</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>150</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_8">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>210</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_9">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>270</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_10">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>340</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableView" name="tableView_2">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>60</y>
         <width>351</width>
         <height>281</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Suppression</string>
      </attribute>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Id de produit :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_idsupp">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>80</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonSupprimer">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>160</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Sms</string>
      </attribute>
      <widget class="QPushButton" name="SendSmsButton">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>220</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Send sms</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Idsms">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>110</y>
         <width>391</width>
         <height>61</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1062</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
